Update trace-perfcollect-lttng.md (#39315)
dotnet/docs: docs/core/diagnostics/trace-perfcollect-lttng.md

Follow these steps to prepare your machine to collect a performance trace with `perfcollect`.

> [!NOTE]

> > If you are capturing from inside a container, your container must have the appropriate capabilities.  The minimal set of capabilities required are `PERFMON` and `SYS_PTRACE`.  If the capture fails with the minimal set, add the `SYS_ADMIN` capability to the container.  For more information on tracing applications inside containers using PerfCollect, see [Collect diagnostics in containers](./diagnostics-in-containers.md).

Reviewer: ```suggestion
> If you're capturing from inside a container, your container must have the appropriate capabilities. The minimal required capabilities are `PERFMON` and `SYS_PTRACE`.  If the capture fails with the minimal set, add the `SYS_ADMIN` capability to the container.  For more information on tracing applications inside containers using PerfCollect, see [Collect diagnostics in containers](./diagnostics-in-containers.md).
```
